$NetBSD: patch-requirements.txt,v 1.1 2019/07/28 16:13:26 nia Exp $

Realistically there's nothing stopping this from working with
Mastodon.py 1.4.6.

--- requirements.txt.orig	2019-07-20 13:29:47.000000000 +0000
+++ requirements.txt
@@ -1,5 +1,5 @@
 click>=6.7
-Mastodon.py==1.4.5
+Mastodon.py>=1.4.5
 colored>=1.3.93
 humanize>=0.5.1
 emoji>=0.4.5
